app/views/components/_taxon-list.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  heading_level ||= 'h2'
+  items ||= []
+%>
+<% if items.any? %>
+  <ol class="app-c-taxon-list">
+    <% items.each_with_index do |item, index| %>
+      <li class="app-c-taxon-list__item">
+        <% if item[:description] %><<%= heading_level %> class="app-c-taxon-list__heading"><% end %>
+        <%=
+          link_to(
+            item[:text],
+            item[:path],
+            data: item[:data_attributes],
+            class: "app-c-taxon-list__link"
+          )
+        %>
+        <% if item[:description] %></<%= heading_level %>><% end %>
+
+        <% if item[:description] %>
```

Comment:
Why are there two duplicate if statements here? Could they be combined?

Reply:
Good point 👍